### DISCUSSION TITLE
is isahc v6 a drop-in replacement for chttp v5?

### DISCUSSION THREAD

**Discussion by ta3pks** [sagebind/isahc#55] [Questions] [answered]
As I understand chttp on crates.io will be no longer maintained since it was stuck on v5 and `isahc` has a v6.
Is it safe to just replace chttp with isahc in Cargo.toml or do i need to do some manual migration due to incompatible api or naming between projects?

**Comment by sagebind** [comment, marked as answer]
There's a couple small breaking changes, but nothing outside of the ordinary for a minor release version. You can see the release notes for 0.6 to see what those changes were. But otherwise, yes! Isahc is a normal continuation of cHTTP, just published under a new name. The chttp crate name on crates.io will be retained for existing projects to use and for historical purposes, but all new releases will be published under the new name.

**Comment by ta3pks** [comment]
thanks

**Comment by ta3pks** [comment]
great to hear that i have checked the changelog seems nothing to effect usual post request

**Comment by sagebind** [comment]
The POST body handling fixes were shipped in the 0.5.4 release, which of course are included in 0.6.0 as well.
